ui/core/components/inputs/consumables.ts

```diff
@@ -353,28 +351,13 @@ export const GiftOfArthas = {
 	actionId: ActionId.fromItemId(9088),
 	value: GuardianElixir.GiftOfArthas,
 };
-export const ElixirOfDraenicWisdom = {
-	actionId: ActionId.fromItemId(32067),
-	value: GuardianElixir.ElixirOfDraenicWisdom,
-};
-export const ElixirOfIronskin = {
-	actionId: ActionId.fromItemId(32068),
-	value: GuardianElixir.ElixirOfIronskin,
-};
-export const ElixirOfMajorDefense = {
-	actionId: ActionId.fromItemId(22834),
-	value: GuardianElixir.ElixirOfMajorDefense,
-};
-export const ElixirOfMajorFortitude = {
-	actionId: ActionId.fromItemId(32062),
-	value: GuardianElixir.ElixirOfMajorFortitude,
-};
-export const ElixirOfMajorMageblood = {
-	actionId: ActionId.fromItemId(22840),
-	value: GuardianElixir.ElixirOfMajorMageblood,
-};
 
 export const GUARDIAN_ELIXIRS_CONFIG = [
+	{ config: ElixirOfDeepEarth, stats: [Stat.StatArmor] },
+	{
+		config: PrismaticElixir,
+		stats: [Stat.StatArcaneResistance, Stat.StatFireResistance, Stat.StatFrostResistance, Stat.StatNatureResistance, Stat.StatShadowResistance],
```

Comment:
Prismatic should give all stats as well right? If so it should then be filtered on those primary stats as well. I dont think a single sim has resistances as an EP value so this poition will not show up on any UI

Reply:
https://www.wowhead.com/cata/spell=80492/prismatic-elixir just the resistances